domain/network/state/space.go

```diff
@@ -332,3 +322,17 @@ func (st *State) DeleteSpace(
 		return nil
 	})
 }
+
+// uniqueSubnetUUIDs returns a deduplicated slice of the Subnet uuids.
+func uniqueSubnetUUIDs(subnets []Subnet) []string {
```

Comment:
We have a package _https://github.com/juju/collections_.

You could probably do this with a one-liner using:
`slice.NewStrings(transform.Slice(subnets <func to get ID>)...).Values`.

Reply:
Added new PR to change this `https://github.com/juju/juju/pull/16836`